fabric_generator/fabric_gen.py

```diff
@@ -582,37 +654,40 @@ def GenerateVHDL_EntityFooter ( file, entity, ConfigPort=True , NumberOfConfigBi
         # file.seek(file.tell() - 3, os.SEEK_SET)     # go backwards 3 bytes
         # file.truncate()
 
-
         print('', file=file)
     print('\t-- global', file=file)
-    if ConfigPort==True:
+    if ConfigPort == True:
```

Comment:
I know this is just automated reformatting, but to note for future cleanup I don't think the `== True` is needed at all